src/components/spots/detail/WriteReview.tsx

```
@@ -0,0 +1,135 @@
+        setIsLoading(false);
+      }
+    })();
+  }, []);
+
+  const [images, setImages] = useState<File[]>([]);
+
+  const previewUrls = images.map((file) => URL.createObjectURL(file));
+
+  useEffect(() => {
+    return () => {
+      previewUrls.forEach((url) => URL.revokeObjectURL(url));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)